tools/column_ordering/column_ordering.rb

```diff
@@ -98,7 +98,7 @@ def table_dump
 
     pg_dump_result = AwesomeSpawn.run("pg_dump", :params => params)
 
-    raise ColumnOrderingError <<-ERROR.gsub!(/^ +/, "") if pg_dump_result.failure?
+    raise ColumnOrderingError(<<-ERROR.gsub!(/^ +/, "")) if pg_dump_result.failure?
```

Comment:
In the case of these, I would say these errors should be made into local variables, and loaded into this method, instead of being defined inside of the raise call:

```ruby
if pg_dump_result.failure?
  err_message = <<-ERROR.gsub!(/^ +/, "")
    '#{pg_dump_result.command_line}' failed with #{pg_dump_result.exit_status}:
    
    stdout: #{pg_dump_result.output}
    stderr: #{pg_dump_result.error}
  ERROR
  raise ColumnOrderingError(err_message)
end
```